config/container_resolvers_conf.xml.sample

```diff
@@ -0,0 +1,35 @@
+<containers_resolvers>
+  <explicit />
+  <!-- explicit: resolves container URI for a job through explict container
+       tags in the tool XML wrapper. -->
+
+  <!-- All mulled flavors below only work if enable_beta_mulled_containers is 
+       set to true in the galaxy.yaml config file.
+
+  <!-- <cached_mulled /> -->
+  <!-- cached_mulled: resolves container URI through bioconda -> mulled
+       automatic mapping, preferring cached containers in the accessible docker
+       engine. Requires docker engine. -->
+
+  <!-- <cached_mulled_singularity /> -->
+  <!-- cached_mulled_singularity: resolves container URI through
+       bioconda -> mulled automatic mapping, preferring cached singularity
+       containers accessible to local singularity binaries. Only works with
```

Comment:
singularity images?

Reply:
as I commented, I'm fairly ignorant in this part, so I'm not sure what would be the correct wording. Please paste here what you would want to see in those pieces of documentation and I can put it on the PR branch.